db/sql/020_create_service_usage_events.sql

```diff
@@ -5,3 +5,5 @@ CREATE TABLE IF NOT EXISTS service_usage_events (
 	raw_message JSONB
 );
 CREATE INDEX IF NOT EXISTS idx_service_usage_id ON service_usage_events (id);
+CREATE INDEX IF NOT EXISTS idx_service_usage_state ON app_usage_events ( (raw_message->>'state') );
+CREATE INDEX IF NOT EXISTS idx_service_usage_type ON app_usage_events ( (raw_message->>'service_instance_type') );
```

Comment:
both indexes should be created on the service_usage_events table

Reply:
Good catch.